### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by clstaudt** [ics-py/ics-py#317] [Ideas]
Since I am working on an open-source project that builds on icy-py, I was wondering what I could to do keep this project well-maintained without involving myself in the development. I would definitely like to sponsor it on GitHub if the option was available.

**Comment by C4ptainCrunch** [comment]
Thank you for your suggestion :)
It's nice to see somebody who would be willing to sponsor our work.

@N-Coder as you are the one that makes most of the work currently, would you be interested to do paid work on ics-py ?
As far as i am concerned, I have a full time job so it's quite hard to spend a lot of time on ics-py as it's one of my (many) side-projects.

@clstaudt Do you know already what amount you would be ready to contribute ? (Send me an email at nikita.marchant@gmail.com, if you don't want to disclose it publicly) Would you ask for specific features in exchange ? Or would this be to keep the lib well maintained ?

**Reply by clstaudt** [reply]
@C4ptainCrunch  Personally I was thinking about small "buy me coffee" kind of money, but monthly as long as my own active projects rely on ics-py. However, more people might be willing to do the same. Therefore, I'm hoping that my suggestion leads you to an evaluation of sponsorship options. In the best case this might free some of your time.  

Concrete example: A missing feature (recurrent events) in ics-py is now a missing feature in my own new open-source project (https://github.com/tuttle-dev). I see the ticket has been open for years, therefore, it would be great if we could free some of your time. 

For example, the new GitHub Sponsors feature seems like a promising way to build an open-source economy.

**Reply by N-Coder** [reply]
While it's really nice to hear this, at least in my case it would probably not help make this project move any faster. What I'm currently lacking is time, not money. I started working on this project for fun while still being a student and now having transitioned to working full time since late 2019 has fastly changed the amount of free time I have at hand (and I guess the same is true for C4ptainCrunch). Furthermore, I guess neither of us is looking to change their employment situation to be able to have some fixed time dedicated for this project, so we probably can't change the current pace being highly dependent on whenever someone has some free time and motivation. Even worse, I guess getting money for this project would make me feel obliged to work more on it without actually freeing up any of my time, meaning either more stress or feeling bad not doing anything in return.
